info/v1/container.go

```diff
@@ -266,25 +266,39 @@ type LoadStats struct {
 // CPU usage time statistics.
```

Comment:
@vishh Ah, I had looked for that actually already. v2 uses the v1 CPU struct and automatically includes my changes: https://github.com/google/cadvisor/blob/master/info/v2/container.go#L136